index.html

```diff
@@ -14,15 +14,21 @@
 <body>
         <header class="header">
             <nav class="nav">
-                <h1><a href="">Kigozi Joseph</a></h1>
+                <h1><a href="https://github.com/zeph180" ref="noreferrer">Kigozi Joseph</a></h1>
                 <i class="fas fa-bars fa-lg"></i>
+                <ul>
```

Comment:
- [ ] As the previous reviewer noted, It would be better to make this hamburger icon display none on the desktop version so that we can still see it on the desktop version since it is supposed to only display on mobile. :blush: 
  ![image](https://user-images.githubusercontent.com/92303891/206879020-50483725-433a-481b-b72a-14e85702e28c.png)